frontend/views/containers/access/SignupForm.vue

```diff
@@ -72,17 +72,17 @@ export default {
         email: null
       },
       ephemeral: {
-        isSubmitting: false
+        isSubmitting: 'false'
```

Comment:
Why did you use a string and not a boolean?

Reply:
Have a look at the Style-Guide.md update in this PR it seems to explain it?

Reply:
Yes, but I'm not sure why we keep this in the dom. I mean I understand that we use [data-loading] has a selector, but that's what I question.

Reply:
We need to keep `data-loading="false"` in the DOM so the spin animation fades in/out properly.

Reply:
Maybe use a class for the button instead of the [data-loading] selector?

Reply:
I tried with classes, but we would need two classes. One to say that is a "loader button" and another to apply the actual loading. 

```
button.button.loader(:class='{"is-loading": isSubmitting"}
```

What do you prefer?

Reply:
How about .loader and use the attribute [data-loading]? The only thing to change is add class loader to the button that have a loader and the boolean attribute will trigger the animation?
Sorry I should have say what I was thinking from the beginning

Reply:
Oh yes, that's an option too. I'll do that then and let you know when it's ready

Reply:
Done! `.is-loader` + `data-loading="Bool"`